docs/data.md

```diff
@@ -125,9 +125,16 @@ log inspection interfaces.
 - `cache.memory.pointers-len` - A gauge representing the number of entries in
   the first level of hashing in the in-memory deduped hashmap.
 
+- `cache.memory.duration-ms` - A duration in milliseconds representing the time 
+  required for removal of expired entries.
+
 - `cache.memory.storage-len` - A gauge representing the number of entries in the
   second level of hashing in the in-memory deduped hashmap.
 
+- `cache.memory.remove-expired` - A record of expired entries for tracing framework 
```

Comment:
Thanks for catching this -- however it's a log message (and of an INFO level) so it belongs in the Logs section above.

Reply:
Thank you! I moved it to the log section, appreciate the semantic clarity, totally missed that